test/run_test.py

```diff
@@ -128,7 +128,8 @@ def skip_test_p(name: str) -> bool:
     ]
 )
 
-TESTS = TESTS + ['doctests']
+# Q: Is there a better way to add "doctests" to the list of tests?
+# TESTS = TESTS + ['doctests']
```

Comment:
I guess, conventionally, there has to be a file. `run_test.py` isn't really setup for virtual tests in this way.

Reply:
If that is the case, and this doesn't map to an exiting pattern, then I think the way I'm doing it here is fine. I added a comment describing why the doctests are a special case.